src/components/navbar/Navbar.component.scss

```diff
@@ -0,0 +1,19 @@
+@import "src/assets/styles/_variables.scss";
+
+.navbar {
```

Comment:
Make sure the classes are scoped. AFAICR React use `.module` system for scoping.